src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -282,9 +282,9 @@ function quad_area (X, Y)
     !  |   |
     !  1 - 2
 
-  p2 = (X(4)-X(1))**2 + (Y(4)-Y(1))**2 ; q2 = (X(3)-X(2))**2 + (Y(3)-Y(2))**2
```

Comment:
If x and y are corresponding directional variables, we might also need to add parentheses around the squares in these 4 expressions, as well as the newly added ones around the whole expressions.